### ISSUE TITLE
Error - finalcutpro/MenuBar.lua

### ISSUE THREAD

**Issue by latenitefilms** [fcpxhacks/fcpxhacks#430]
See: 

```
2017-04-08 22:11:30: ********
2017-04-08 22:11:30: 22:11:30 ERROR:   LuaSkin: hs.chooser:choices error - ...ub/CommandPost/src/extensions/cp/finalcutpro/MenuBar.lua:211: attempt to index a nil value (local 'menu')
stack traceback:
	...ub/CommandPost/src/extensions/cp/finalcutpro/MenuBar.lua:211: in function 'cp.finalcutpro.MenuBar._visitMenuItems'
	(...tail calls...)
	.../CommandPost/src/plugins/finalcutpro/menu/menuaction.lua:121: in field 'choices'
	...s/Github/CommandPost/src/plugins/core/action/manager.lua:343: in field '_findChoices'
	...s/Github/CommandPost/src/plugins/core/action/manager.lua:334: in function <...s/Github/CommandPost/src/plugins/core/action/manager.lua:332>
	(...tail calls...)
	[C]: in method 'refreshChoicesCallback'
	.../CommandPost/src/plugins/finalcutpro/console/console.lua:142: in function <.../CommandPost/src/plugins/finalcutpro/console/console.lua:141>
2017-04-08 22:11:30: ********
2017-04-08 22:11:30: ********
2017-04-08 22:11:30: 22:11:30 ERROR:   LuaSkin: hs.chooser:choices error - ...ub/CommandPost/src/extensions/cp/finalcutpro/MenuBar.lua:211: attempt to index a nil value (local 'menu')
stack traceback:
	...ub/CommandPost/src/extensions/cp/finalcutpro/MenuBar.lua:211: in function 'cp.finalcutpro.MenuBar._visitMenuItems'
	(...tail calls...)
	.../CommandPost/src/plugins/finalcutpro/menu/menuaction.lua:121: in field 'choices'
	...s/Github/CommandPost/src/plugins/core/action/manager.lua:343: in field '_findChoices'
	...s/Github/CommandPost/src/plugins/core/action/manager.lua:327: in function <...s/Github/CommandPost/src/plugins/core/action/manager.lua:325>
	[C]: in method 'refreshChoicesCallback'
	.../CommandPost/src/plugins/finalcutpro/console/console.lua:143: in function <.../CommandPost/src/plugins/finalcutpro/console/console.lua:141>
2017-04-08 22:11:30: ********
```

**Comment by randomeizer** [comment]
I believe this was fixed in #306. Can you confirm if it's still occurring?

**Comment by randomeizer** [comment]
I've made some additional improvements to checking if the menu exists. Merged.
